src/pile.js

```diff
@@ -1019,6 +1023,36 @@ const createPile = (
     setCover(newCover);
   };
 
+  const placeholderGfx = new PIXI.Graphics();
+  let isPlaceholderDrawn = false;
+
+  const drawDot = (x, y, r) => {
+    placeholderGfx.lineStyle(0);
+    placeholderGfx.beginFill(0xffffff, 1);
```

Comment:
The color should be adjusted based on whether `store.state.darkMode` is `true` or `false`